pftools/python/parflow/tools/ref/well_keys.yaml

```diff
@@ -0,0 +1,198 @@
+# -----------------------------------------------------------------------------
```

Comment:
Right now we can't support all the well keys because some of the subkeys such as the cycle are dynamically named. Talked with Reed and he said these need to be reworked anyways so to get in what is supported